docs/features.md

```diff
@@ -3,4 +3,30 @@ title: Features
 order: 1900
 ---
 
-#Features
+# Scully's Features
+
+Under the hood, Scully analyzes your Angular application and generates a static
+version of it. It provides several Angular schematics to make
+its usage AS EASY AS POSSIBLE!
+
+[Core Features](/docs/scully-lib-core):
+
+- Idle Monitor Service
+- Router Service
+- Scully Content Component
+- Transfer State Service
+- Utility Methods
+
+[Plugins System](/docs/plugins):
+
+- Router Plugins
+- Render Plugins
+- File Handler Plugins
+
```

Comment:
missing `alldone` and `routeDiscoveryDone` plugins.
Both are called with the complete routelist, and their result is ignored.